src/groupByKey.ts

```diff
@@ -2,6 +2,27 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+interface Item {
+  'id': number,
+  'color': string,
+  'country': string,
+}
+
+export function groupByKey(
+  items: Item[],
+  key: keyof Item,
+): GroupsMap<Item> {
```

Comment:
```suggestion
export function groupByKey<T>(
  items: T[],
  key: keyof T,
): GroupsMap<T> {
```

Reply:
I've changed my solution. Please watch it once more. Thx